assets/product-info.js

```diff
@@ -91,13 +93,34 @@ if (!customElements.get('product-info')) {
         this.#renderProductInfo(targetUrl, variant?.id, targetId, callback);
       }
 
+      // Load the product page and update sections
+      #updateProductSections(url, variantId) {
+        this.sectionRefreshAbortController?.abort();
+        this.sectionRefreshAbortController = new AbortController();
+        fetch(this.#getProductInfoUrl(url, variantId, true), {
+          signal: this.sectionRefreshAbortController.signal,
+        })
+          .then((response) => response.text())
+          .then((responseText) => {
+            const html = new DOMParser().parseFromString(responseText, 'text/html');
+            const updatedSections = Array.from(html.querySelectorAll('main > .shopify-section')).reduce(
+              (sectionsById, section) => ({ ...sectionsById, [section.id]: section }),
+              {}
+            );
+
+            document.querySelectorAll('main > .shopify-section').forEach((section) => {
+              if (!section.id.includes('__main') && updatedSections[section.id]) {
```

Comment:
@tyleralsbury / @tauthomas01 I'm not sure how section IDs are generated.... is it a safe assumption to make that the section IDs will match between the initial product and the sibling products?